README.md

```diff
@@ -13,6 +13,14 @@ Then most rules have been elevated into warnings, but lowered some to suggestion
 1. Copy `.editorconfig` into the root of your repository
 2. Copy `Directory.Build.Props` into the root of your repository (this will treat warnings as errors in Release mode - typically your CICD pipeline) 
 
+### Script
+
+Alternatively, copy & paste this script in a Terminal, inside the root directory of your project:
+
+```powershell
+$editorConfigPath="./.editorconfig";if(-not (Test-Path $editorConfigPath)){Invoke-WebRequest "https://raw.githubusercontent.com/SSWConsulting/SSW.EditorConfig/main/.editorconfig" -OutFile $editorConfigPath;Write-Host "File installed: '$editorConfigPath'"}else{Write-Host "File already exists: '$editorConfigPath'"};$directoryBuildPropsPath="./Directory.Build.Props";if(-not (Test-Path $directoryBuildPropsPath)){Invoke-WebRequest "https://raw.githubusercontent.com/SSWConsulting/SSW.EditorConfig/main/Directory.Build.Props" -OutFile $directoryBuildPropsPath;Write-Host "File installed: '$directoryBuildPropsPath'"}else{Write-Host "File already exists: '$directoryBuildPropsPath'"}
```

Comment:
Is there anyway we can have one version of this script that's referenced in both the readme and the PS file?

Reply:
Done - script now downloads & invokes from install.ps1.

Tested by modifying url for my branch file.